frontend/chhattisgarhi-explorer/chhattisgarhi.css

```
@@ -0,0 +1,420 @@
+/* chhattisgarhi.css — Chhattisgarhi Language Explorer */
+
+.devanagari-title,
+.greeting-devanagari,
+.word-devanagari {
+    font-family: "Noto Sans Devanagari", "Outfit", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)